src/robusta/core/sinks/slack/slack_sink_params.py

```
@@ -1,18 +1,47 @@
 from robusta.core.sinks.sink_base_params import SinkBaseParams
 from robusta.core.sinks.sink_config import SinkConfigBase
 from robusta.core.sinks.common import ChannelTransformer
-
-from typing import Optional
+from enum import Enum
+from typing import Optional, Dict, Literal
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)